microprofile/src/main/fabric8/example-microprofile-svc.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: example-microprofile
+  annotations:
+    description: Inventory Service based on WF Swarm
+spec:
+  ports:
+    - protocol: TCP
+      port: 8080
+      targetPort: 8080
+  selector:
+    deploymentconfig: example-microprofile
+  type: LoadBalancer
```

Comment:
Should it be `ClusterIP`? You have a `Route` above, so don't need a `LoadBalancer` service, right?